14-pvc-deployment/README.md

```diff
@@ -80,7 +80,7 @@ And make sure that data was persisted across restarts
 ## Application Clean-Up
 
 ```bash
-kubectl delete -f .
+kubectl delete ns doit-lab-14
```

Comment:
Maybe we should delete the  pv, pvc beforehand to avoid resources left behind? (disks)

Reply:
The default reclaim policy for PVCs is `Delete` [1], it was cleaned out as the PVC was deleted [2].

[1] https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/#why-change-reclaim-policy-of-a-persistentvolume
[2] https://cloudlogging.app.goo.gl/q2k6sLqA265XLpHfA